app/controllers/EventDetailsController.js

```diff
@@ -1,8 +1,10 @@
 GameUpApp.controller('EventDetailsController', ['$scope', '$location', 'eventService', 'events', function($scope, $location, eventService, events){
+
+  var TicketObject = Parse.Object.extend("Tickets");
   var eventId = $location.path().replace('/events/', '');
 
   $scope.eventObj = null;
```

Comment:
Does this need to be initialized to null? I feel like this isn't necessary, however this could be a style thing.